cvat-ui/src/containers/annotation-page/standard-workspace/controls-side-bar/draw-shape-popover.tsx

```diff
@@ -67,6 +70,7 @@ function mapStateToProps(state: CombinedState, own: OwnProps): StateToProps {
 type Props = StateToProps & DispatchToProps;
 
 interface State {
```

Comment:
Let's add this value also to global redux state.
We need it in order to in the future be able to start drawing using only one shortcut, with all previous settings.

Reply:
I see. I added activeRectDrawingMethod in [cvat-ui/src/reducers/annotation-reducer.ts](https://github.com/opencv/cvat/pull/1127/files#diff-385a09313d2054c79c235452ff60070bR359)